.coafile

```diff
@@ -40,7 +40,7 @@ ignore_length_regex = ^.*https?://
 [DOCS]
 bears = SpaceConsistencyBear, LineLengthBear
 
-files = docs/**/*.rst, CONTRIBUTING.rst, README.rst
+files = **.rst
```

Comment:
@Vamshi99 can you please have a look at this one. And suggest me how to proceed furthur with the issues that are coming.